packages/components/text-field/src/text-field.ts

```diff
@@ -285,6 +285,7 @@ export class TextField<T extends { toString(): string } = string>
     this.input.focus();
   }
 
+  /** @internal */
```

Comment:
Do we really want to hide these methods? They're `protected` for a reason: so people can override them :)